src/domain/Exercise.java

```diff
@@ -1,7 +1,22 @@
 package domain;
 
+/**
+ * Class that callculates palindrome
+ * @author Artem
+ */
 public class Exercise {
-    public static float Calculate(int R){
-        return (float) (2*Math.PI*Math.sqrt(R));
+    
+    /**
+     * Method returns whether the string is a palindrome
+     * @param string string that will be check
+     * @return boolean(true: if string is palindrome, false: if string isn`t palindrome)
+     */
+    public static boolean IsPalindrome(String string){

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.